examples/text-generation/README.md

```diff
@@ -619,6 +619,76 @@ python run_generation.py \
 --flash_attention_recompute
 ```
 
+### Saving FP8 Checkpoints in Hugging Face format
+After quantizing the model, we can save it to a local path.
+
+> [!NOTE]  
+> Before executing the command below, please refer to the "Running FP8 Models on a Single Device" section to measure the model quantization statistics.
```

Comment:
```suggestion
> Before executing the command below, please refer to the ["Running FP8 Models on a Single Device" section](#running-fp8-models-on-a-single-device) to measure the model quantization statistics.
```
Let's add a link to it too, it will be easier for readers.

Can you also add the same here please? https://github.com/HabanaAI/optimum-habana-fork/blob/98539470b6c6dbf39145844317d878ddf5482167/examples/text-generation/README.md?plain=1#L600
I had missed that.